src/js/Snackbars/Snackbar.js

```diff
@@ -157,11 +157,13 @@ export default class Snackbar extends PureComponent {
       };
 
       if (typeof action !== 'string') {
-        btnProps = Object.assign(btnProps, action, {
+        btnProps = Object.assign(btnProps, {
```

Comment:
I think it would be better to do

```
btnProps = Object.assign(btnProps, action, {
  className: cn(btnProps.className, action.className),
  onClick: this._handleClick,
});
```

instead of this. The main reason for merging them together was so that you could override any of the default props for the button